app/src/main/res/values/dimens.xml

```diff
@@ -48,6 +48,9 @@
 
     <dimen name="default_fab_button_size">56dp</dimen>
 
+    <dimen name="step_divider_width">138dp</dimen>
+    <dimen name="layout_padding_12dp">12dp</dimen>
```

Comment:
Build fails with this error :- "Error: Found item Dimension/layout_padding_12dp more than one time"
```suggestion
```